src/core/loaders/Loader.ts

```diff
@@ -200,6 +202,27 @@ export abstract class Loader extends Disposable {
     return str
   }
 
+  private treeNodeValueHasPluralizationKeys(value: Record<string, any>, locale: string) {
+    if (!value)
+      return false
+
+    return isObject(value)
+      && this.pluralizationKeysV4.some((key: string) => get(value, key) != null)
+  }
+
+  private firstPluralizationKeyValue(value: Record<string, any>, locale: string) {
+    if (!value)
```

Comment:
Just in case this method isn't called alongside `treeNodeValueHasPluralizationKeys`

```suggestion
    if (!value || !isObject(value))
```